generators/app/templates/src/app/components/CustomTextInput/index.js

```diff
@@ -7,71 +7,64 @@ import { transparent } from '@constants/colors';
 import ShowPassword from './components/ShowPassword';
 import styles from './styles';
 
-// Known issue: showing/hiding the text with secureTextEntry changes the cursor position
-// https://github.com/facebook/react-native/issues/5859
+const CustomTextInput = props => {
+  const [showPassword, setShowPassword] = useState(false);
 
-class CustomTextInput extends PureComponent {
-  state = { showPassword: false };
+  const handleShowPassword = useCallback(() => setShowPassword(prevShowPassword => !prevShowPassword), []);
```

Comment:
I'm not sure if you should pass `[setShowPassword]` as dependency to the useCallback

Reply:
According to the docs "React guarantees that setState function identity is stable and won’t change on re-renders. This is why it’s safe to omit from the useEffect or useCallback dependency list." https://github.com/reactjs/reactjs.org/blob/master/content/docs/hooks-reference.md